en/docs/how-tos/support-custom-claims-mapping.md

```diff
@@ -1,14 +1,20 @@
 # Support Custom Claims Mapping
 
-When JWT retrieved from Multiple Identity providers, Microgateway can map the relevant claims as a supported one to validate the JWT.  The claims of the incoming authentication JWT can differ based on the Authorization server that issued the JWT token. This feature maps the incoming JWT remote claims to the local claims of the Microgateway as well as the claims expected by the back end service.  You can use custom claims mapping transformation in micro gateway when you have different keys or values of claims in your JWT token to make it as a Microgateway supported one.
+Microgateway provides the capability to map any claims in an incoming JWT token to user specified claims using custom claims mapping. 
 
-In a detailed view for an example if you generate JWT with claims with a different key like claim key is ‘scp’ and value is ‘[“write”, “read”]’. But in Microgateway it only validates correctly if the key is scope and values of scope separated by space as a string. Hence through the custom claims mapping feature, you can map the remote claims to local claims to support the Microgateway.  Here remote claims come from an external provider thus local claims need to be mapped.
- 
-1. If the key of the remote claims is different from the custom claims, then you need to change the configuration as below.  You can define the multiple keys of the remote claim and local claim.
+When JWT tokens are retrieved from multiple identity providers, Microgateway can map the relevant claims to supported claims in order to validate the JWT. The claims of the incoming authentication JWT can differ based on the Authorization server that issued the JWT token. This feature maps the incoming JWT remote claims to the local claims of the Microgateway as well as the claims expected by the back end service. You can use custom claims mapping transformation in Microgateway when you have different keys or values of claims in your JWT token to make it into a Microgateway supported one.
 
-     Adding the following section to the micro-gw.conf file which is located in the <MGW-RUNTIME-HOME>/conf directory, you can change the configurations in the jwtTokenConfig. Here the remote claim is scp and the local claim is scope.
-#### Configuring claims key
-```
+For example, if you generate a JWT token with the claim `"scp": [“write”, “read”]` to represent the scope of the token, since Microgateway validates scopes only when they are attached to the token with the claim key 'scope' and when the scopes are separated by spaces as a string. Therefore, a custom claim mapping can be used to transform the remote claim to a local claim.
```

Comment:
```suggestion
For example, if you generate a JWT token with the claim `"scp": [“write”, “read”]` to represent the scope of the token, since Microgateway validates scopes only when they are available in the token with the claim key 'scope' and when the scopes are separated by spaces as a string. Therefore, a custom claim mapping can be used to transform the remote claim to a local claim.
```